enaml/core/parser.py

```diff
@@ -3445,6 +3445,56 @@ def p_arglist14(p):
     p[0] = Arguments(keywords=keywords, starargs=p[2], kwargs=p[8])
 
 
+def _validate_arglist_and_kwlist(p, items, keywords):
```

Comment:
Would you mind adding a comment about the assumption for the line number? That way we can update if we ever refactor this code.